acceptance/src/test/java/io/github/tcdl/msb/acceptance/bdd/util/ListAppender.java

```diff
@@ -0,0 +1,53 @@
+package io.github.tcdl.msb.acceptance.bdd.util;
+
+import org.apache.log4j.AppenderSkeleton;
+import org.apache.log4j.spi.LoggingEvent;
+import org.junit.Assert;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class ListAppender extends AppenderSkeleton {
```

Comment:
Now we have ListAppender in msb-java, so there's no need to duplicate it in microservices.

__Please remove it from dam and ghcm microservices if you have time__